docs/src/pages/landing/components/toggle.tsx

```diff
@@ -0,0 +1,26 @@
+/** @jsxImportSource react */
+const Toggle: React.FC<{
+  id: string;
+  label: string;
+  onSelectionChange?: (selected: boolean) => void;
+}> = ({ id, label, onSelectionChange = (_: boolean) => {} }) => {
+  return (
+    <div>
+      <span className="mr-4">{label}</span>
+      <div className="switch focus-within:outline-black">
```

Comment:
we want to show the outline when necessary
```suggestion
      <div className="switch focus-within:outline-black focus-within:outline">
```